ext/devcontainer/src/azd/README.md

```diff
@@ -2,16 +2,26 @@
 
 Installs the [Azure Developer CLI](https://github.com/Azure/azure-dev) along with needed dependencies.
 
-## Example Usage
+## Example Usage - Install latest `azd` version
 
 ```json
 "features": {
-    "ghcr.io/azure/azure-dev/azd:1": {
-        "version": "stable"
-    }
+    "ghcr.io/azure/azure-dev/azd:latest": {}
```

Comment:
Just wondering - could we tag this with `1` so that the import could be:

```
"ghcr.io/azure/azure-dev/azd:1"
```

That would be more in line with how other features seem to be tagged?